x-pack/plugins/fleet/server/services/package_policies_to_agent_permissions.ts

```diff
@@ -98,6 +98,7 @@ export async function storedPackagePoliciesToAgentPermissions(
                     type: stream.data_stream.type,
                     dataset:
                       stream.compiled_stream?.data_stream?.dataset ?? stream.data_stream.dataset,
+                    allow_routing: stream.data_stream.allow_routing,
```

Comment:
If this is going to give privileges on data streams outside of the ones of this package, shall we warn the user when this privilege is given? Otherwise any package could use this setting to inadvertently obtain too much privileges.

Reply:
Sounds like a good idea. Do you have a suggestion on how and were to warn the user?

Reply:
I guess that this would need to be done when installing a package with this flag enabled.